common/rpc/rpc.go

```diff
@@ -229,6 +232,14 @@ func (d *RPCFactory) CreateLocalFrontendGRPCConnection() *grpc.ClientConn {
 
 // CreateInternodeGRPCConnection creates connection for gRPC calls
 func (d *RPCFactory) CreateInternodeGRPCConnection(hostName string) *grpc.ClientConn {
+	connection := d.interNodeGrpcConnections.Get(hostName)
+	if connection != nil {
+		c, ok := connection.(*grpc.ClientConn)
+		if !ok {
+			panic("unexpected entry in interNodeGrpcConnections cache")
+		}
+		return c
```

Comment:
you can just do
```suggestion
		return connection.(*grpc.ClientConn)
```
it will panic if it's the wrong type, but you're panicking anyway so it's the same thing

Reply:
Linter is complaining in that case. It is silly that if I add explicit panic, it does not have a problem. Maybe it wants to make it explicit in the code

Reply:
I have refactored it.